x-pack/test/alerting_api_integration/observability/custom_threshold_rule/avg_pct_fired.ts

```diff
@@ -38,8 +37,19 @@ export default function ({ getService }: FtrProviderContext) {
     // DATE_VIEW should match the index template:
     // x-pack/packages/kbn-infra-forge/src/data_sources/composable/template.json
     const DATE_VIEW = 'kbn-data-forge-fake_hosts';
-    const DATE_VIEW_NAME = 'data-view-name';
+    const DATE_VIEW_NAME = 'ad-hoc-data-view-name';
     const DATA_VIEW_ID = 'data-view-id';
+    const MOCKED_AD_HOC_DATA_VIEW = {
+      id: DATA_VIEW_ID,
+      title: DATE_VIEW,
+      timeFieldName: '@timestamp',
+      sourceFilters: [],
+      fieldFormats: {},
+      runtimeFieldMap: {},
+      allowNoIndex: false,
+      name: DATE_VIEW_NAME,
+      allowHidden: false,
+    };
```

Comment:
might be worth using this util from data plugin `createStubDataView`

Reply:
**Update**

I tried it but got this error:

![image](https://github.com/elastic/kibana/assets/12370520/a7f112db-16cb-4469-9e6c-d6f4fa8d379b)